Error passing tests building the developer box - scala agent / vm err

**Describe the bug**
After trying to build a new ODS Box, based in the scripts provided '/ods-core/ods-devenv/scripts/run-on-aws.sh' based in the branch feature/aqua-configmaps, that was created based in the master branch, the suite of tests that is executed finalize failing

Nothing has been changed related with the test that had been executed.

**To Reproduce**
Steps to reproduce the behavior:
1. Checkout the code
2.  execute $./run-on-aws.sh --install --target-git-ref feature/aqua-configmaps

**Expected behavior**
The box is build and everything works as expected.


**Affected version (please complete the following information):**
 - OpenShift: [3.11]
 - OpenDevStack [branch from master]

**Log Output (ensure to remove any confidential information like tokens, project names, etc.**
```
Verifying OpenShift resources of nginx-iq-test in unitt-dev ...
--- FAIL: TestQuickstarter (0.00s)
    --- PASS: TestQuickstarter/be-fe-mono-repo-plain (263.86s)
    --- PASS: TestQuickstarter/fe-ionic (747.33s)
    --- PASS: TestQuickstarter/release-manager (838.00s)
    --- FAIL: TestQuickstarter/ods-provisioning-app (91.76s)
    --- FAIL: TestQuickstarter/ods-document-gen-svc (71.76s)
    --- PASS: TestQuickstarter/docker-plain (203.09s)
    --- PASS: TestQuickstarter/fe-angular (466.73s)
    --- PASS: TestQuickstarter/e2e-spock-geb (326.54s)
    --- PASS: TestQuickstarter/e2e-cypress (286.62s)
    --- PASS: TestQuickstarter/ds-rshiny (1344.94s)
    --- PASS: TestQuickstarter/ds-jupyter-lab (383.61s)
    --- PASS: TestQuickstarter/be-python-flask (246.60s)
    --- PASS: TestQuickstarter/be-typescript-express (386.72s)
    --- PASS: TestQuickstarter/be-scala-play (466.97s)
    --- PASS: TestQuickstarter/be-golang-plain (246.55s)
    --- PASS: TestQuickstarter/be-java-springboot (386.81s)
    --- PASS: TestQuickstarter/be-gateway-nginx (223.20s)
FAIL
FAIL    github.com/opendevstack/ods-core/tests/quickstarter     6981.104s
```

We would like to go live  in July, could you give us some hints of what could happen @clemensutschig , @michaelsauter ?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -34,6 +34,7 @@
 - ds-rshiny not able to deploy in OCP 4 ([#609](https://github.com/opendevstack/ods-quickstarters/issues/609))
 - fixed mixed line endings on multiple files ([#618](https://github.com/opendevstack/ods-quickstarters/issues/618))
 - fix dead sbt rpm bintray repo ([#622](https://github.com/opendevstack/ods-quickstarters/issues/622))
+- openjdk 11 does not recognize VM setting ([#623](https://github.com/opendevstack/ods-quickstarters/issues/623))
 
 ### Removed
 

--- a/common/jenkins-agents/scala/docker/Dockerfile.rhel7
+++ b/common/jenkins-agents/scala/docker/Dockerfile.rhel7
@@ -16,6 +16,10 @@ RUN yum install -y java-11-openjdk-devel && \
     javac -version
 ENV JAVA_HOME=/usr/lib/jvm/jre
 
+# Container support is now integrated in Java 11, the +UseCGroupMemoryLimitForHeap option has been pruned
+ENV JAVA_TOOL_OPTIONS="-XX:+UnlockExperimentalVMOptions -Dsun.zip.disableMemoryMapping=true"
+
+
 # Install sbt
 ENV SBT_VERSION=1.5.4
 ENV SBT_CREDENTIALS="$HOME/.sbt/.credentials"

--- a/common/jenkins-agents/scala/docker/Dockerfile.ubi8
+++ b/common/jenkins-agents/scala/docker/Dockerfile.ubi8
@@ -16,6 +16,9 @@ RUN yum install -y --enablerepo centos-appstream java-11-openjdk-devel && \
     javac -version
 ENV JAVA_HOME=/usr/lib/jvm/jre
 
+# Container support is now integrated in Java 11, the +UseCGroupMemoryLimitForHeap option has been pruned
+ENV JAVA_TOOL_OPTIONS="-XX:+UnlockExperimentalVMOptions -Dsun.zip.disableMemoryMapping=true"
+
 # Install sbt
 ENV SBT_VERSION=1.5.4
 ENV SBT_CREDENTIALS="$HOME/.sbt/.credentials"

